crates/ecs/src/lib.rs

```diff
@@ -1265,9 +1294,18 @@ mod tests {
     fn components_are_stored_on_correct_index_independent_of_insert_order() {
         let mut archetype = Archetype::default();
 
-        let entity_1_id = 0;
-        let entity_2_id = 10;
-        let entity_3_id = 5;
+        let entity_1 = Entity {
+            id: 0,
+            _generation: 0,
+        };
```

Comment:
Looks like a constructor `Entity::with_id(n)` would be handy? If this'll be unnecessary in the very next PR then you can skip this, otherwise it could be nice to have.

Reply:
I'll place it in the test module, since it will not be used anywhere else